src/components/search-result-list/search-result-list-item/search-result-list-item-cover.tsx

```diff
@@ -6,16 +6,19 @@ const SearchResultListItemCover: React.FC<SearchResultListItemCoverProps> = ({
   id,
   description,
   url,
-  tint
+  tint,
+
+  linkAriaLabelledBy
 }) => {
   return (
     <Cover
       animate
       id={id}
       size="small"
-      description={String(description)}
+      description={description}
```

Comment:
Please consider if both `description` and `linkAriaLabelledBy` is appropriate here.

The description (which will become an `alt` value if provided) *and* `linkAriaLabelledBy` serves the same purpose. I understand that the `Cover` component accepts both but as I see it for the `SearchResultListItemCover` we only want to provide one of them: `linkAriaLabelledBy`.